flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/spec/AbstractFlinkSpec.java

```diff
@@ -40,4 +42,7 @@ public abstract class AbstractFlinkSpec {
      * restart, change the number to anything other than the current value.
      */
     private Long restartNonce;
+
+    /** Flink configuration overrides for the Flink deployment. */
```

Comment:
nit: the comment should be update: overrides for the Flink deployment or session job.